turborepo-tests/helpers/setup_example_test.sh

```diff
@@ -55,24 +55,28 @@ mkdir -p ./tmp
 echo "/tmp/" >> ".gitignore"
 
 # Simulating the user's first run and dumping logs to a file
+$package_manager --version >./tmp/package_manager_version.txt
 $package_manager_command >./tmp/install.txt 2>&1
-$turbo_command >./tmp/grep-me-for-miss.txt
+$turbo_command >./tmp/run-1.txt
```

Comment:
Expected rename?

Reply:
yes: I was trying to make it clear.  we're running the same command twice so the output for run 1 goes to `run-1.txt` and the output for run 2 goes to `run-2.txt`.  At least for me, this added clarity to understanding what's going on.  The `grep-me` thing was cute, but I found it hard to follow upon trying to understand what's going on here without any other context or asking anyone.